batctl/Makefile

```diff
@@ -122,6 +122,7 @@ config-n := \
 	log \
 	loglevel \
 	mcast_flags \
+	multicast_forceflood \
 	multicast_mode \
```

Comment:
Comment says:

> replace multicast_mode with multicast_forceflood settings subcommand

Here, you ADD and do not replace. Haven't checked the code. Is this intentional?

Reply:
Yes, this is expected. We don't want to break backward compatibility yet. You will not see the old command in the help but it is still accepted.

The PR itself has just the changelogs from the release.